From 622527cc6d2c27ef2afe1b5adfb18a3b6656d57a Mon Sep 17 00:00:00 2001
From: pafonsoX <philippex.afonso@intel.com>
Date: Wed, 22 Jan 2014 10:50:56 +0100
Subject: [PATCH 4/9] [REVERT ME][AUDIO] Implement LPAL source in policy

BZ: 217971

Orig-Bug: 166294

This patch implements the definition of Low-Power
Always Listening (LPAL) audio source.
The audio source LPAL is low priority.

Change-Id: I04d48fbb3194258bf05c1f0fcd52e0257fdc99de
Category: feature differentiation
Domain: AudioComms-VoiceTrigger
Origin: internal
Upstream-Candidate: no, proprietary
Signed-off-by: pafonsoX <philippex.afonso@intel.com>
---
 include/hardware_legacy/AudioSystemLegacy.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/hardware_legacy/AudioSystemLegacy.h b/include/hardware_legacy/AudioSystemLegacy.h
index eaaa982..bcd4c24 100644
--- a/include/hardware_legacy/AudioSystemLegacy.h
+++ b/include/hardware_legacy/AudioSystemLegacy.h
@@ -62,7 +62,7 @@ enum audio_source {
     AUDIO_SOURCE_VOICE_RECOGNITION = 6,
     AUDIO_SOURCE_VOICE_COMMUNICATION = 7,
     AUDIO_SOURCE_MAX = AUDIO_SOURCE_VOICE_COMMUNICATION,
-
+    AUDIO_SOURCE_LPAL = 2000,
     AUDIO_SOURCE_LIST_END  // must be last - used to validate audio source type
 };
 
-- 
1.9.2

